extension-manifest-v3/src/utils/api.js

```diff
@@ -11,14 +11,20 @@
 
 import jwtDecode from 'jwt-decode';
 
-const DOMAIN = 'ghostery.com';
+const DOMAIN = 'ghosterystage.com';
```

Comment:
Reminder: before we merge, we should change it (and the production server should be updated).

Reply:
This is not intentional, I was undoing that change before committing. I'll fix it.